PlaceRouteHierFlow/placer/SeqPair.cpp

```diff
@@ -1173,18 +1173,18 @@ std::string SeqPair::getLexIndex(design& des) const {
 bool SeqPair::CheckSymm(design& caseNL) {
   auto logger = spdlog::default_logger()->clone("placer.SeqPair.CheckSymm");
   std::map<int, int> posPosition, negPosition;
-  for (int i = 0; i < posPair.size(); ++i) {
+  for (int i = 0; i < ((int)posPair.size()); ++i) {
     posPosition[posPair[i]] = i;
     negPosition[negPair[i]] = i;
   }
   for (const auto& sb : caseNL.SBlocks) {
 
     // self symm blocks should be (above/below for vertical axis) or (left/right for horizontal axis)
     // self symm blocks to the (left/right for vertical axis) or (above/below) for horizontal) is a violation
-    for (int i = 0; i < sb.selfsym.size() - 1; ++i) {
+    for (int i = 0; i < ((int)sb.selfsym.size()) - 1; ++i) {
```

Comment:
This is where the problem is, subtracting one from an unsigned number. The other way to fix this is to change the loop variable to one more than `i` and start the loop at 1.